### PR TITLE
Update locale-es.json

### PR DESCRIPTION
**Pull request by Cryptopeppe**
#804

### CODE REVIEW

**Comment by Cryptopeppe** [comment]
my address cryptopeppe3

File: app/assets/locales/locale-es.json [hunk 1]
```diff
@@ -8,6 +17,16 @@
     "es": "Español",
     "it": "Italiano",
     "tr": "Türkçe"
+    "ru": "Русский"
```
Comment (svk31) [inline app/assets/locales/locale-es.json]: There's a comma missing here.

File: app/assets/locales/locale-es.json [hunk 2]
```diff
@@ -95,6 +183,15 @@
       "subscribe": "Comprar Suscripción Anual",
       "expires": "expira",
       "membership_expiration": "Vencimiento de la Membresía"
+      "fees_cashback": "Estadísticas de comisìon",
```
Comment (svk31) [inline app/assets/locales/locale-es.json]: Another comma missing

File: app/assets/locales/locale-es.json [hunk 3]
```diff
@@ -125,27 +272,59 @@
...
+      "password_model": "Modelo de cuenta",
+      "password_model_1": "Si desea poder iniciar sesión desde cualquier lugar solo con su contraseña, puede usar esta página para agregar nuevas claves a sus permisos. Si también quieres eliminar las claves originales, puede hacerlo en las pestañas de permisos de propietario/activo. ",
+      "password_model_2": "Agregar claves generadas por contraseña",
+      "new_active": "Nueva clave activa",
+      "new_owner": "Nueva clave de propietario",
+      "new_memo": "Nueva clave memo",
+      "memo_warning": "ADVERTENCIA: Si reemplaza la tecla memo no podrá leer las notas viejas cuando inicie la sesión con su contraseña, y a menos que importe la nueva clave privada memo en su billetera anterior esa billetera no podrá decodificar nuevos memos",
+      "use_text": "Utilizar",
+      "remove_text": "Eliminar"
+       "valid_pub": "Llàve publica valida"
```
Comment (svk31) [inline app/assets/locales/locale-es.json]: comma again

File: app/assets/locales/locale-es.json [hunk 4]
```diff
@@ -190,39 +407,56 @@
...
+    "eq_value": "Estimated",
+    "no_price": "--",
+    "percent": "Porcentaje de oferta total",
     "please_create_account": "Por favor cree una cuenta",
     "create_account": "Crear cuenta",
+    "create_new": "Crear una nueva cuenta",
+    "password_login": "Iniciar la sesión con contraseña",
     "identicon": "Identicon",
-    "pay_from": "Pagar Desde",
+    "pay_from": "Pagar con la cuenta,
```
Comment (svk31) [inline app/assets/locales/locale-es.json]: Missing closing quote

File: app/assets/locales/locale-es.json [hunk 5]
```diff
@@ -190,39 +407,56 @@
       "invalid": "Nombre de cuenta no válido",
       "unknown": "Cuenta desconocida",
       "not_yours": "No es el propietario de la cuenta"
+           "memo_missing": "No tiene control de la tecla de nota para esta cuenta. Para utilizar una nota, configure la tecla de nota de cuenta en una tecla que controle utilizando la página Permisos."
```
Comment (svk31) [inline app/assets/locales/locale-es.json]: comma

**Comment by Cryptopeppe** [comment]
Updated, it should be ok!

**Pull request merged**